### PR TITLE
[Feature][Connector-V2][RDS] add rds doc

### PR DESCRIPTION
**Pull request by s7monk**
@https://github.com/apache/incubator-seatunnel/issues/2829

## Purpose of this pull request
add rds doc

## Check list

* [X] Code changed are covered with tests, or it does not need tests for reason:
* [X] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [X] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [X] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)

### CODE REVIEW

**Comment by Hisoka-X** [comment]
Please refer Sink/Greenplum.md to update. Add some special JDBC config description for this connector.
And add it into appendix
image

**Comment by s7monk** [comment]
> Please refer Sink/Greenplum.md to update. Add some special JDBC config description for this connector. And add it into appendix image

ok

File: docs/en/connector-v2/source/Jdbc.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ there are some reference value for params above.
 | starrocks  | com.mysql.cj.jdbc.Driver                            | jdbc:mysql://localhost:3306/test                                       | https://mvnrepository.com/artifact/mysql/mysql-connector-java                                               |
 | db2        | com.ibm.db2.jcc.DB2Driver                           | jdbc:db2://localhost:50000/testdb                                      | https://mvnrepository.com/artifact/com.ibm.db2.jcc/db2jcc/db2jcc4                                           |
 | tablestore | com.alicloud.openservices.tablestore.jdbc.OTSDriver | "jdbc:ots:http s://myinstance.cn-hangzhou.ots.aliyuncs.com/myinstance" | https://mvnrepository.com/artifact/com.aliyun.openservices/tablestore-jdbc                                  |
-
+| rds        | com.mysql.cj.jdbc.Driver                            | jdbc:mysql://localhost:3306/test                                       | https://mvnrepository.com/artifact/mysql/mysql-connector-java                                               |
 ## Example
```
Comment (TaoZex) [inline docs/en/connector-v2/source/Jdbc.md:114]: Add a blank line before Example.

**Review by TaoZex — COMMENTED** [review]
In general, we only need to update the jdbc documentation.

File: docs/en/connector-v2/source/Jdbc.md (hunk 1 above)
Comment (s7monk) [reply docs/en/connector-v2/source/Jdbc.md:114]: ok

File: docs/en/connector-v2/sink/Jdbc.md [hunk 2]
```diff
@@ -132,6 +132,7 @@ there are some reference value for params above.
 | GBase8a    | com.gbase.jdbc.Driver                        | jdbc:gbase://e2e_gbase8aDb:5258/test                               | /                                                  | https://www.gbase8.cn/wp-content/uploads/2020/10/gbase-connector-java-8.3.81.53-build55.5.7-bin_min_mix.jar |
 | StarRocks  | com.mysql.cj.jdbc.Driver                     | jdbc:mysql://localhost:3306/test                                   | /                                                  | https://mvnrepository.com/artifact/mysql/mysql-connector-java                                               |
 | db2        | com.ibm.db2.jcc.DB2Driver                    | jdbc:db2://localhost:50000/testdb                                  | com.ibm.db2.jcc.DB2XADataSource                    | https://mvnrepository.com/artifact/com.ibm.db2.jcc/db2jcc/db2jcc4                                           |
+| rds        | com.mysql.cj.jdbc.Driver                     | jdbc:mysql://localhost:3306/test                                   | com.mysql.cj.jdbc.MysqlXADataSource                | https://mvnrepository.com/artifact/mysql/mysql-connector-java                                               |
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Jdbc.md:135]: Please add changed log.

File: docs/en/connector-v2/sink/Rds.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Rds
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Rds.md:1]: remove this doc?

File: docs/en/connector-v2/source/Jdbc.md [hunk 4]
```diff
@@ -109,6 +109,7 @@ there are some reference value for params above.
 | starrocks  | com.mysql.cj.jdbc.Driver                            | jdbc:mysql://localhost:3306/test                                       | https://mvnrepository.com/artifact/mysql/mysql-connector-java                                               |
 | db2        | com.ibm.db2.jcc.DB2Driver                           | jdbc:db2://localhost:50000/testdb                                      | https://mvnrepository.com/artifact/com.ibm.db2.jcc/db2jcc/db2jcc4                                           |
 | tablestore | com.alicloud.openservices.tablestore.jdbc.OTSDriver | "jdbc:ots:http s://myinstance.cn-hangzhou.ots.aliyuncs.com/myinstance" | https://mvnrepository.com/artifact/com.aliyun.openservices/tablestore-jdbc                                  |
+| rds        | com.mysql.cj.jdbc.Driver                            | jdbc:mysql://localhost:3306/test                                       | https://mvnrepository.com/artifact/mysql/mysql-connector-java                                               |
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/Jdbc.md:112]: Please add changed log.

File: docs/en/connector-v2/source/Rds.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# Rds
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/Rds.md:1]: remove this doc?

**Comment by CalvinKirs** [comment]
TBH, I don't know what the point of this PR is. RDS uses other database engines, in principle, as long as we support the corresponding engines.

**Pull request closed without merging**